internal/app/go-filecoin/internal/submodule/storage_protocol_submodule.go

```diff
@@ -138,3 +149,19 @@ func (sm *StorageProtocolSubmodule) PieceManager() (piecemanager.PieceManager, e
 	}
 	return sm.pieceManager, nil
 }
+
+func (sm *StorageProtocolSubmodule) DataTransfer() datatransfer.Manager {
```

Comment:
This and the following `RequestValidator` don't appear to be used. If they are needed outside this submodule it would be better to move data transfer into the networking submodule. Otherwise, they should be removed.